### PR TITLE
Bug 1314763 - Enable mozlog logging format and django-configurations

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by jezdez** [comment]
@maurodoglio Note, some of the config values in your `.env` need to be renamed:

- `DEBUG` ➡️ `DJANGO_DEBUG`
- `SITE_URL` ➡️ `DJANGO_SITE_URL`
- `ALLOWED_HOSTS ` ➡️  `DJANGO_ALLOWED_HOSTS`

File: atmo/settings.py [hunk 1]
```diff
@@ -13,369 +13,357 @@
...
         }
-    }
+        revision = atmo.get_revision()
+        if revision:
+            config['release'] = revision
+        return config
+
+
+class Prod(Stage):
+    """Configuration to be used in stage environment"""
```
Comment (maurodoglio) [inline atmo/settings.py]: These docstrings refer to Stage and viceversa

File: atmo/settings.py [hunk 2]
```diff
@@ -13,369 +13,357 @@
...
+        'django.contrib.auth.hashers.MD5PasswordHasher',
+    )
 
-SECURE_BROWSER_XSS_FILTER = True
-SECURE_CONTENT_TYPE_NOSNIFF = True
-SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
-X_FRAME_OPTIONS = 'DENY'
 
-if SITE_URL.startswith('https://'):
+class Stage(Base):
```
Comment (maurodoglio) [inline atmo/settings.py]: s/Stage/Staging ?

Comment (jezdez) [inline atmo/settings.py]: Sadly, the dockerflow uses `stage` instead: https://atmo.stage.mozaws.net

**Pull request merged**